src/campaign/sass/components/_hero-banner.scss

```diff
@@ -265,27 +265,11 @@ $hero-text-colour: #fff;
 }
 
 .hero-banner__background--apprentice-hosanna{
-  background-image: linear-gradient(255deg, rgba(0,0,0,0) 0%, rgba(0,0,0,1) 100%), url("../images/apprentice/hozanna-hero-1.jpg");
-  .hero-banner--right-aligned & {
```

Comment:
As per slack message, im not sure we should be removing these styles as i feel all the images should be all be the same alignment